doc/man/scons.xml

```diff
@@ -2930,28 +2930,19 @@ include:</para>
 
 <para>In addition to the global functions and methods,
 &scons;
-supports a number of Python variables
+supports a number of variables
 that can be used in SConscript files
-to affect how you want the build to be performed.
```

Comment:
this bit was moved to the end of the section as it's not the main subject of the section